genesis/manifests/examples/core.element.yaml

```diff
@@ -0,0 +1,367 @@
+        controller:
+          kind: systemd
+          service:
+            kind: daemon
+            exec_start: gc-user-api --config-file /etc/genesis_core/genesis_core.conf
+        depends_on:
+          - $core.element.services.$migrations:get_uri()
+      
+      orch_api:
+        name: Genesis Core Orch Api
+        description: Genesis Core Orch API service
+        target:
+          kind: node
+          node: $core.compute.nodes.$bootstrap:uuid
+        controller:
+          kind: systemd
+          service:
+            kind: daemon
+            exec_start: gc-orch-api --config-file /etc/genesis_core/genesis_core.conf
+        depends_on:
+          - $core.element.services.$migrations:get_uri()
+        
+      core_agent:
+        name: Genesis Core Agent
+        description: Genesis Core Agent
+        target:
+          kind: node
+          node: $core.compute.nodes.$bootstrap:uuid
+        controller:
+          kind: systemd
+          service:
+            kind: daemon
+            exec_start: gc-gservice --config-file /etc/genesis_core/genesis_core.conf
+        depends_on:
+          - $core.element.services.$migrations:get_uri()
+
+
+  $core.iam.users:
+    admin:
+      uuid: "00000000-0000-0000-0000-000000000000"
+      username: "admin"
+      description: "System administrator"
+      first_name: "Admin"
+      last_name: "User"
+      surname: ""
+      email: "admin@example.com"
+
+
+  $core.iam.organizations:
+    admin:
+      uuid: "00000000-0000-0000-0000-000000000000"
+      name: "admin"
+      description: "Admin Organization"
+    genesis_corporation:
+      uuid: "11111111-1111-1111-1111-111111111111"
+      name: "Genesis Corporation"
+      description": "The organization serves as the central platform for all services and elements developed by Genesis Corporation."
+
+
+  $core.iam.organization_members:
+    admin:
+      uuid: "210a146e-0ddd-4021-85cd-51760d9b0cbd"
+      organization: "$core.iam.organizations.$admin:get_uri()" 
+      user: "$core.iam.users.$admin:get_uri()"
+      role: "OWNER"
+  
+
+  $core.iam.projects:
+    admin:
+      uuid: "00000000-0000-0000-0000-000000000000"
+      name: "admin"
+      description: "Admin Project"
+      organization: $core.iam.organizations.$admin:get_uri()
+    iam_core:
+      uuid: "c938820c-ab9c-56e4-919c-acd0e881996f"
+      name: "iam-core"
+      description: "Identity and Access Management Core Project"
+      organization: $core.iam.organizations.$genesis_corporation:get_uri()
+    compute:
+      uuid: "46f57877-f477-5916-bcd1-718d34cf3414"
+      name: "compute-core"
+      description: "Comptue and Baremetal Core Project"
+      organization: "$core.iam.organizations.$genesis_corporation:get_uri()"
+  
+
+  $core.iam.permissions:
+    admin:
+      uuid: "00000000-0000-0000-0000-000000000000"
+      name: "*.*.*"
+      description: "Allow All"
+
+    iam_user_list:
+      uuid: "a8a7eea2-7514-53ac-a408-8b1235352310"
```

Comment:
offtopic - maybe use uuidv5 to get consistent uuids automatically for permissions?

Reply:
this uuid is from migrations file